src/index.html

```diff
@@ -1,15 +1,165 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="POST"
+          autocomplete="off"
+          name="form"
+          onsubmit="onSubmit()">
+      <fieldset class="info_area">
+        <legend>Personal information</legend>
+        <div class="info">
+          <label for="surname">Surname:
+            <input type="text"
+                   id="surname"
+                   name="surname">
+          </label>
+        </div>
+        <div class="info">
+          <label for="name">Name:
+            <input type="text"
+                   id="name"
+                   name="name">
+          </label>
+        </div>
+        <div class="info">
+          <label for="age">
+            How old are You?
+            <input type="number"
+                   id="age"
+                   name="age"
+                   value="12"
+                   min="1"
+                   max="100">
+          </label>
+        </div>
+        <div class="info">
+          <label for="date">Full date of birth:
+            <input type="date"
+                   id="date"
+                   name="date">
+          </label>
+        </div>
+        <div class="info">
+          <label for="agreement">I accept the term of agreement
+            <input type="checkbox"
+                   id="agreement"
+                   name="agreement">
+          </label>
```

Comment:
```suggestion
          <label for="agreement">
            I accept the term of agreement
            <input 
              type="checkbox"
              id="agreement"
              name="agreement"
            >
          </label>
```
отредактируй в соотвествии с этим примером